aip/general/0111.md

```diff
@@ -0,0 +1,93 @@
+---
+id: 111
+state: draft
```

Comment:
When can we change it to approved?...before submitting? :D

Love this guidance